doc/api/tls.md

```diff
@@ -770,6 +774,19 @@ stream.
 
 NOTE: `cleartext` has the same API as [`tls.TLSSocket`][]
 
+**Deprecated** `tls.createSecurePair()` is now deprecated in favor of
+`tls.TLSSocket()`. For example:
+```
```

Comment:
style nit, can you add the `js` to the end of the three backticks and put a blank line before the examples